### PR TITLE
Add introduction to open source

### LINKED ISSUE #4: Write "Introduction to Open Source"
see hints in the Resource issue #3

### PR DESCRIPTION
**Pull request by bobturneruk**
resolves #4

### CODE REVIEW

File: content/01-contributing-oss/slides/index.en.md
Comment (drj11) [inline content/01-contributing-oss/slides/index.en.md:30]: but a PR does now need to be approved by the repo maintainer to count (this is new).

File: content/01-contributing-oss/slides/index.en.md
Comment (drj11) [inline content/01-contributing-oss/slides/index.en.md]: If the BBC Data Unit has a history of sharing Open Source, I couldn't find it. I spotted checked the first four repos mentioned here: https://github.com/BBC-Data-Unit/shared-data-unit . None were published with a LICENCE.

Personally, I don't really care, but this article specifically mentions licensing.

File: content/01-contributing-oss/slides/index.en.md
Comment (drj11) [inline content/01-contributing-oss/slides/index.en.md]: There are some direct costs that are unarguably lower. An example: renting cloud compute on AWS is more expensive if you choose Windows, a proprietary operating system, over Linux, a Open Source one.

File: content/01-contributing-oss/slides/index.en.md
Comment (drj11) [inline content/01-contributing-oss/slides/index.en.md:51]: I would add: find collaborators.

File: content/01-contributing-oss/slides/index.en.md
Comment (drj11) [inline content/01-contributing-oss/slides/index.en.md]: VS Code might be a good example: https://github.com/microsoft/vscode (since you also mention it later in tools)

File: content/01-contributing-oss/slides/index.en.md
Comment (drj11) [inline content/01-contributing-oss/slides/index.en.md:123]: Good examples!

**Review by drj11 — APPROVED** [review]
Looks good. I'm pressing "Submit review" before I build this branch, but I'll also try that.

As ever, exercise you skill and judgement when considering if my comments need changes. I'd be entirely happy if it went in unchanged.

File: content/01-contributing-oss/slides/index.en.md
Comment (bobturneruk) [reply content/01-contributing-oss/slides/index.en.md:30]: Is this so - I thought it would be rejected if marked as spam, but does not need approval to count?
image
Although my profile looks different!
image
I'm confused! Is there something definitive you've found somewhere, please?

File: content/01-contributing-oss/slides/index.en.md
Comment (bobturneruk) [reply content/01-contributing-oss/slides/index.en.md]: Cheers - I inherited this - will remove for simplicity's sake for now.

File: content/01-contributing-oss/slides/index.en.md
Comment (bobturneruk) [reply content/01-contributing-oss/slides/index.en.md]: Thanks. Really good point. I will also bring up that you don't get support so something looks like it's free but you still need to install, patch, maintain, document...

File: content/01-contributing-oss/slides/index.en.md
Comment (bobturneruk) [reply content/01-contributing-oss/slides/index.en.md:51]: Thanks!

File: content/01-contributing-oss/slides/index.en.md
Comment (bobturneruk) [reply content/01-contributing-oss/slides/index.en.md]: Great idea.

File: content/01-contributing-oss/slides/index.en.md
Comment (bobturneruk) [reply content/01-contributing-oss/slides/index.en.md:123]: Thanks!

**Comment by bobturneruk** [comment]
Thanks for your input @drj11 - I will look into the issue of whether things need to be approved (I'm sure you're right). I won't merge for now as @annakrystalli may want to make some suggestions, as the originator of this content, if time allows.

File: content/01-contributing-oss/slides/index.en.md
Comment (davidwyld) [reply content/01-contributing-oss/slides/index.en.md:30]: This is a VERY recent development https://hacktoberfest.digitalocean.com/hacktoberfest-update
As it says, PR's will only count if accepted by one of those 3 methods listed.

Comment (drj11) [reply content/01-contributing-oss/slides/index.en.md:30]: yeah i thought it was in the digital ocean blog i read this morning. I'm talking about un-accepted PRs (accepted PRs in a hacktoberfest tagged repo automatically count, I think).

Comment (bobturneruk) [reply content/01-contributing-oss/slides/index.en.md:30]: Ta!

Comment (bobturneruk) [reply content/01-contributing-oss/slides/index.en.md:30]: I've linked to the new rules from the talk.

Comment (ptheywood) [reply content/01-contributing-oss/slides/index.en.md:30]: I believe PRs opened prior to the changes being made by hacktoberfest will count without having to follow the new rules (hence your two exisitng PRs are `In Review` rather than `Pending`

**Comment by drj11** [comment]
I looked at the compiled slides, and the only thing worth commenting was on slide 9 the "note to self" is on the slide. If that's intentional, that's fine.

**Comment by bobturneruk** [comment]
Yes, thanks, intentional. My memory forces this.

**Pull request merged**
